__tests__/endpoints.test.js

```diff
@@ -0,0 +1,47 @@
+const data = require('../db/data/test-data/index')
+const request = require('supertest')
+const app = require('../app')
+const seed = require('../db/seeds/seed')
+const db = require('../db/connection')
+
+
+beforeEach(() => {
+    return seed(data)
+})
+
+afterAll(() => {
+    return db.end()
+})
+
+describe('Test Endpoints', () => {
+    describe('/api/topics', () => {
+        test('200 - fetch all the topics from db', () => {
+            return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.length).toBe(3);
+            })
+        })
+        it("GET 404 Endpoint does not exist", () => {
+            return request(app)
+            .get('/api/trees')
+            .expect(404)
+            .then((response) => {
+              expect(response.status).toBe(404)
+              expect(response.body.msg).toBe('URL Not Found');
+            })
+        });
+    });
+    describe('/api', () => {
+        test('200 - fetch available endpoints', () => {
+            return request(app)
+            .get("/api")
+            .expect(200)
+            .then(({body}) => {
+                expect(body).toBeInstanceOf(Object);
+                expect(body['GET /api']).toHaveProperty('description');
```

Comment:
when testing endpoints here you should expect the object you receive from the response to be similar to the object inside your endpoints file so I would require in read the endpoints file here and the expect it to be similar to endpoints from the response.